docs/user/user/integrating-glean-for-product-managers.md

```diff
@@ -0,0 +1,45 @@
+# Integrating Glean for product managers
+
+This chapter provides guidance for planning the work involved in integrating Glean into your product, for internal Mozilla customers.
+For a technical coding perspective, see [adding Glean to your project](adding-glean-to-your-project.html).
+
+Glean is the standard telemetry platform required for all new Mozilla products.
+While there are some upfront costs to integrating Glean in your product, this pays off in easier long-term maintenance and a rich set of self-serve analysis tools.
+The Glean SDK team is happy to support your telemetry integration and make it successful.
+Find us in [#glean](https://chat.mozilla.org/#/room/#glean:mozilla.org) or email [glean-team@mozilla.com](mailto:glean-team@mozilla.com).
+
+## Building a telemetry plan
+
+The Glean SDK provides support for answering basic product questions out-of-the-box, such as daily active users, product version and platform information.  
+However, it is also a good idea to have a sense of any additional product-specific questions you are trying to answer with telemetry, and, when possible, in collaboration with a data scientist.  
+This of course helps for your own planning, but is also invaluable for the Glean SDK team to support you, since we will understand the ultimate goals of your product's telemetry and ensure the design will meet those goals and we can identify any new features that may be required.
+It is best to frame this document in the form of questions and use cases rather than as specific data points and schemas.
+
+## Integrating the Glean SDK into your product
+
+The [technical steps for integrating the Glean SDK in your product are documented elsewhere](adding-glean-to-your-project.html) for supported platforms.
+We recommend having a member of the Glean SDK team review this integration to catch any potential pitfalls.
+
+## (Optional) Adapting Glean to your platform
+
+The Glean SDK is a collection of cross platform libraries and tools that facilitate collection of Glean conforming telemetry from applications.  
+Consult the list of the currently supported [platforms and languages](../index.html).
```

Comment:
The next best thing we have is the [List of implementations](https://mozilla.github.io/glean/dev/core/internal/implementations.html).
Do we have a bug filed to create something more user-focused?

Reply:
Just made one: https://bugzilla.mozilla.org/show_bug.cgi?id=1708429